chicago/bills.py

```diff
@@ -2,12 +2,14 @@
 import lxml
 import lxml.etree
 
-from pupa.scrape import Bill
+from pupa.scrape import Bill, Vote
 
 
 class ChicagoBillScraper(LegistarScraper):
     base_url = 'https://chicago.legistar.com/'
     legislation_url = 'https://chicago.legistar.com/Legislation.aspx'
+    timezone = "US/Central"
+
 
     def searchLegislation(self, search_text='', created_before=None,
```

Comment:
Does pupa have any facilities for passing through command line args. It would be great to be able to specify search parameters.